### DISCUSSION TITLE
Good first issue levels

### DISCUSSION THREAD

**Discussion by AdityaPimpalkar** [twentyhq/twenty#1598] [General]
I believe introducing different levels of issue complexity for new contributors would be a fantastic idea. It can motivate newcomers to participate and provide a clear progression path as well. We could have labels like (Good first issue: Level 1) and so on.. 

Thoughts on this?

**Comment by FelixMalfait** [comment]
Yes great idea, I was thinking about something similar yesterday and started created new labels: "blocked", "type", "size", "scope"
The issue with "level 1" or "level 2" is that it's subjective. 
What do you think of the new labels? do they help or not?

Size is mostly correlated with difficulty but not always... I'm not sure if it's worth introducing a fifth label...
We could also have a label for prioritization. But I feel like the list of issues is already hard to read so it will start looking really bad if we have 6+ label per issue 😅

**Reply by AdityaPimpalkar** [reply]
Yes! The type, size and scope are pretty self explanatory 👍 
"blocked: false" is a little confusing, a simple "ready" would be great I think, just an idea 😄 

Haha, I agree with issues having 6+ labels, it looks very overwhelming.

**Reply by FelixMalfait** [reply]
haha yes I named it this way for consistency but agree it's confusing. Just changed it to "ready", thanks for the feedback!
